### PR TITLE
fix(acl): Check timeout

### PR DESCRIPTION
**Pull request by Guillaume28**
#5547

### CODE REVIEW

File: www/class/centreonContactgroup.class.php [hunk 1]
```diff
@@ -211,9 +211,9 @@ public function syncWithLdap()
                     /*
                      * Test is the group a not move or delete in ldap
                      */
-                    if (false === $ldapConn->getEntry($row['cg_ldap_dn'])) {
+                    if (false === $ldapConn->getEntry($row['cg_ldap_dn']) && ldap_errno($ldapConn->getDs()) !=3) {
```
Comment (leoncx) [inline www/class/centreonContactgroup.class.php]: Missing space between the = and 3

File: www/class/centreonContactgroup.class.php [hunk 2]
```diff
@@ -211,9 +211,9 @@ public function syncWithLdap()
...
                         $dn = $ldapConn->findGroupDn($row['cg_name']);
-                        if (false === $dn) {
+                        if (false === $dn  && ldap_errno($ldapConn->getDs()) !=3) {
```
Comment (leoncx) [inline www/class/centreonContactgroup.class.php]: Missing space between the = and 3

**Comment by ganoze** [comment]
Does it have to be merged on 2.8.x and master branches too ?

**Comment by Guillaume28** [comment]
It seem's yes, the same problem appear in the 2.8.x

**Pull request merged**